src/axom/sidre/docs/sphinx/group.rst

```diff
@@ -84,14 +87,15 @@ Group objects.
             become invalid. This is analogous to iterator invalidation for
             containers when the container contents change.
 
-Create and Destroy Groups
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create, Modify, and Destroy Groups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
  * Create a child Group given a name (child) or path (other descendant). 
    If a path is given, intermediate Groups in path are created, if needed. 
- * Rename a Group.
+ * Rename a Group.  A Group cannot be renamed to the empty string or to
+   a string containing the path separator character.
```

Comment:
or to the name of another Group or View owned by the same parent.

Reply:
@agcapps it would be good to add the point of clarification that @nselliott pointed out.

Reply:
Thanks @nselliott ---done